tests/test_unit.py

```diff
@@ -345,9 +346,13 @@ def test_run_poppunk_internal(qtbot):
     # wait for assign job to be finished
     def assign_status_finished():
         job = Job.fetch(job_ids["assign"], connection=redis)
-        assert job.get_status() == "finished"
+        try:
+            assert job.get_status() == "finished"
+        except AssertionError:
+            return False
+        return True
```

Comment:
Does this need an assert? Coudn't it just do
```suggestion
       return job.get_status() == "finished" 
```